frontend/probloom/nginx.conf

```diff
@@ -0,0 +1,19 @@
+server {
+  listen 80;
+  location / {
+    root /usr/app/build;
+    index index.html index.htm;
+    try_files $uri $uri/ /index.html;
+  }
+  location /api {
+    proxy_pass http://3.34.29.178:8000/api;
+  }
+  location /static {
+    root /usr/app/build;
+    try_files $uri @django;
+  }
+  location @django {
+    proxy_pass http://3.34.29.178:8000;
+  }
```

Comment:
This part differs from the sample configuration in our practice session. I added this part because we might be able to add image insertion features in our service.